packages/libs/react-ui/src/components/NavHeader/NavHeader.stories.children.tsx

```diff
@@ -0,0 +1,17 @@
+import { Button } from '@components/Button';
+import React, { FC } from 'react';
+
+export const NavHeaderChildren: FC = () => {
+  return (
+    <Button
+      as="button"
+      icon="Link"
+      onClick={() => {}}
```

Comment:
I feel like this is unfinished?

Reply:
It's not. It's just sample content.

Reply:
ahh yeah, I missed that this was in a story 👍